src/app/modules/consultations/consultation-profile/profile-card/profile-card.component.ts

```diff
@@ -117,16 +117,38 @@ export class ProfileCardComponent implements OnInit, OnChanges {
       let { diffInDays, isSameDay } = this.getDifferenceInDays(deadline);
       diffInDays = Math.floor(diffInDays);
 
+      const translations = {
```

Comment:
Same here

Reply:
@gpalsingh - The translations in this function are dynamically generated based on the diffInDays value, which is calculated at runtime. Since phrases like "X Days Remaining" depend on this value, it’s not practical to store them as static entries in translation.json

Reply:
@Bandinikhil - how many strings exactly and why isn't it practical?